### PR TITLE
test: add the necessary judgment

### PR DESCRIPTION
**Pull request by renhwztetecs**
1. test/librados: add ASSERT_EQ & ASSERT_NE judgment
2. test/libcephfs: add the necessary judgment

Signed-off-by: huanwen ren <ren.huanwen@zte.com.cn>

### CODE REVIEW

**Comment by yuriw** [comment]
@renhwztetecs this PR seems to be failing in "[  FAILED  ] LibRadosIo.ReadTimeout" test
Run: http://pulpito.ceph.com/yuriw-2016-07-19_15:14:36-rados-wip-yuri-testing__2016_7_18-distro-basic-smithi/

323179, 323216, 323239, 323243

One for example in http://qa-proxy.ceph.com/teuthology/yuriw-2016-07-19_15:14:36-rados-wip-yuri-testing__2016_7_18-distro-basic-smithi/323179/teuthology.log
2016-07-19T15:44:18.334 INFO:tasks.workunit.client.0.smithi019.stdout:                   api_io: /srv/autobuild-ceph/gitbuilder.git/build/rpmbuild/BUILD/ceph-11.0.0/src/test/librados/io.cc:57: Failure
2016-07-19T15:44:18.335 INFO:tasks.workunit.client.0.smithi019.stdout:                   api_io: Expected: (0) != (rados_connect(cluster)), actual: 0 vs 0
2016-07-19T15:44:18.335 INFO:tasks.workunit.client.0.smithi019.stdout:                   api_io: [  FAILED  ] LibRadosIo.ReadTimeout (3165 ms)
``

File: src/test/librados/io.cc [hunk 1]
```diff
@@ -50,12 +50,12 @@ TEST_F(LibRadosIo, ReadTimeout) {
...
-    rados_conf_read_file(cluster, NULL);
-    rados_conf_parse_env(cluster, NULL);
-    rados_conf_set(cluster, "rados_osd_op_timeout", "0.00001"); // use any small value that will result in a timeout
-    rados_connect(cluster);
-    rados_ioctx_create(cluster, pool_name.c_str(), &ioctx);
+    ASSERT_EQ(0, rados_create(&cluster, "admin"));
+    ASSERT_EQ(0, rados_conf_read_file(cluster, NULL));
+    ASSERT_EQ(0, rados_conf_parse_env(cluster, NULL));
+    ASSERT_EQ(0, rados_conf_set(cluster, "rados_osd_op_timeout", "0.00001")); // use any small value that will result in a timeout
+    ASSERT_NE(0, rados_connect(cluster));
```
Comment (jdurgin) [inline src/test/librados/io.cc]: this should succeed - rados_osd_op_timeout only applies to OSD ops (like the read checked below), not the initial connection

Comment (renhwztetecs) [inline src/test/librados/io.cc]: thanks
I ,missing  change it

**Comment by renhwztetecs** [comment]
Done

**Comment by jdurgin** [comment]
thanks, looks good

**Comment by renhwztetecs** [comment]
@yuriw 
help me testing
thank you

**Comment by tchaikov** [comment]
tested http://pulpito.ceph.com/kchai-2016-08-18_15:35:36-rados-wip-kefu-testing2---basic-mira/

all failures are known ones.

File: src/test/librados/io.cc (hunk 1 above)
Comment (tchaikov) [inline src/test/librados/io.cc]: @renhwztetecs. seems we still have this change? see http://tracker.ceph.com/issues/17087

Comment (tchaikov) [inline src/test/librados/io.cc]: fixed by #10806

Comment (renhwztetecs) [inline src/test/librados/io.cc]: I am sure I modified it，I don't know what happened
But thank you to fixup it

**Pull request merged**